compiler/rustc_codegen_llvm/src/type_of.rs

```diff
@@ -20,23 +20,16 @@ fn uncached_llvm_type<'a, 'tcx>(
     match layout.abi {
         Abi::Scalar(_) => bug!("handled elsewhere"),
         Abi::Vector { ref element, count } => {
+            // FIXME: Replace this comment
```

Comment:
I think we can just remove this comment. The previous MMX hack would likely have caused issues with `packed_simd` anyways.

Reply:
Like [that one](https://github.com/rust-lang/rust/issues/53254) affecting windows-gnu and apple?